### ISSUE TITLE
Please use git tags for releases

### ISSUE THREAD

**Issue by marmistrz** [emscripten-core/emsdk#463]
This would make it much easier to package emsdk for distributions (in this case, for Arch Linux)

**Comment by kripken** [comment]
Do you mean in the emsdk repo itself? We do tag the other repos, see https://github.com/emscripten-core/emscripten/blob/master/docs/process.md#minor-version-updates-1xy-to-1xy1

**Comment by marmistrz** [comment]
Yes, I mean the emsdk repo itself.

**Comment by kripken** [comment]
Yeah, that sounds reasonable too, good idea!

I opened https://github.com/emscripten-core/emscripten/pull/10869 to update the process doc, and I pushed tags for the last 10 or so releases to this repo now.

**Comment by sbc100** [comment]
Seems reasonable.  I guess we could go back and look at each change to `emscripten-releases-tags.txt` and tag that.  Basically all changed `emscripten-releases-tags.txt` introduce new revisions.

emsdk is a bit different in that it support all older SDK versions too.  It doesn't just represent the latest version, but I guess there is harm tagging X.Y.Z as the first revision that introduced X.Y.Z.

**Comment by kripken** [comment]
Yeah, it is slightly different that the emsdk supports old versions too. But makes sense that if someone wants version X, the tag where it was added is a good commit for it.

**Comment by sbc100** [comment]
This was fixed a long time ago now.
